samples/Config.CustomConfigurationProviders.Web/project.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "*.config ConfigurationProviders Library",
+  "authors": [ "Microsoft" ],
+  "tags": [ "configuration" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
+  "content": [
+    "Web*.config"
+  ],
+
+  "dependencies": {
+    "Microsoft.Extensions.Configuration": "1.0.0-rc1-final",
```

Comment:
I think tis is what you're looking for: https://github.com/aspnet/Configuration/blob/70b9b0325be999e80bc97dfda095b73e0e17c790/src/Microsoft.Extensions.Configuration.Abstractions/ConfigurationPath.cs#L14

The key point is that you can't mix versions of packages (e.g. RC1 and RC2), they always have to be a "coherent set," so either everything needs to be hard-coded to a specific version, or everything needs to float together. In the case of the repos here we never hard-code the version because we always want it to compile against the most recent builds.